app/views/ama_layout/_notification_sidebar.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="off-canvas position-right right-sidebar" id="offCanvasRight" data-off-canvas>
+  <div class="row column right-sidebar__header">
+    <h2 class="mt1">Notification Centre</h2>
+    <button class="close-button" aria-label="Close menu" type="button" data-close>
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="row column">
+    <%= navigation.notifications_heading %>
+  </div>
+  <%= render partial: 'ama_layout/notification', collection: notifications %>
+  <div class="row column">
+    <p class="mt1">Did You Know?</p>
+    <div class="section-content">
+      <p class="bold">Renew Your Membership Automatically</p>
```

Comment:
I'm working with the PO/business unit and it's looking like this is going to switch to a VRAR promotion instead.